specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2025-04-01/containerregistry.json

```diff
@@ -3105,11 +3084,8 @@
       ],
       "properties": {
         "properties": {
-          "allOf": [
-            {
-              "$ref": "#/definitions/ConnectedRegistryProperties"
-            }
-          ],
+          "$ref": "#/definitions/ConnectedRegistryProperties",
```

Comment:
Why do we need this change in connected registry resources? I don't see the change in other resources.